bin/generate-gutenboarding-design-thumbnails.js

```diff
@@ -13,29 +13,60 @@
  * GENERATE THUMBNAILS:
  *
  * `node ./bin/generate-gutenboarding-design-thumbnails.js`
+ * or
+ * `node ./bin/generate-gutenboarding-design-thumbnails.js all`
+ * `node ./bin/generate-gutenboarding-design-thumbnails.js mag16`
+ * `node ./bin/generate-gutenboarding-design-thumbnails.js es`
+ * `node ./bin/generate-gutenboarding-design-thumbnails.js fr,de,zh-cn`
  *
  * We should bump the version query param on the image src to cache bust the images too in client/landing/gutenboarding/available-designs.ts
  *
  * To check the results, take a look at http://calypso.localhost:3000/new/design and the output folder static/images
  *
  */
 
-const captureWebsite = require( 'capture-website' );
+const fs = require( 'fs' );
 const sharp = require( 'sharp' );
+const puppeteer = require( 'puppeteer' );
 const wpUrl = require( '@wordpress/url' );
+
 const designs = require( '../client/landing/gutenboarding/available-designs-config.json' );
-const screenshotsPath = './static/images/design-screenshots'; // Folder to store output images
+const config = require( '../client/server/config/index.js' );
+
+const mag16 = config( 'magnificent_non_en_locales' ) || [];
+const screenshotsBasePath = './static/images/design-screenshots'; // Folder to store output images
 
 // image output variables
-const captureMaxHeight = 2200; // Cap long pages to  this pixel height when capturing
+const captureMaxHeight = 3072; // Cap long pages to  this pixel height when capturing
 const outputWidth = 900; // Outputted file width
 const viewportHeight = 800; // Browser height for capturing the screenshot
 const viewportScaleFactor = 2; // Browser pixel density for capturing the screenshot
-const viewportWidth = 1280; // Browser width for capturing the screenshot
+const viewportWidth = 900; // Browser width for capturing the screenshot
```

Comment:
Potentially a good idea to match the [new calypso](https://github.com/Automattic/wp-calypso/blob/0e3a272e1446247cd98d583d5f817ce0a1398b4a/client/landing/gutenboarding/available-designs.ts#L53) values for portrait previews: `{ vpw: 1600, vph: 3000, w: 600, h: 1124 }`

Or at least the ratios maybe - I had a quick try with this but got an error I didn't understand about composite sizing :grimacing: